apprise/plugins/NotifyBase.py

```diff
@@ -424,4 +425,5 @@ def parse_url(url, verify_host=True):
         if 'user' in results['qsd']:
             results['user'] = results['qsd']['user']
 
+        results['headers'] = {k[1:]: v for k, v in results['qsd'].items() if re.match(r'^-.', k)}
```

Comment:
This is great man; so simple and clean!  This line is the only one I'd like you to tidy up a wee-bit.  PEP8 will complain that it's too long (>79 characters).

Just changing it to something like this should fix it:
```python
# new line + indent of 4 spaces
results['headers'] = {
    k[1:]: v for k, v in results['qsd'].items() if re.match(r'^-.', k)}
```